test/test_common/simulated_time_system.h

```diff
@@ -105,8 +115,9 @@ class SimulatedTimeSystemHelper : public TestTimeSystem {
   RealTimeSource real_time_source_; // Used to initialize monotonic_time_ and system_time_;
   MonotonicTime monotonic_time_ ABSL_GUARDED_BY(mutex_);
   SystemTime system_time_ ABSL_GUARDED_BY(mutex_);
+  Runtime::RandomGeneratorImpl random_source_ ABSL_GUARDED_BY(mutex_);
```

Comment:
driveby: if you use TestRandom it'll cerr the seed for reproducability and allow flag based override.

Reply:
Thanks, done.